packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -201,12 +203,16 @@ export class Viewer {
     }
 
     if (this.cameraControls) {
-      this.registerInteractionHandler(new MouseInteractionHandler());
-      this.registerInteractionHandler(new TouchInteractionHandler());
+      if (!this.usePointerEvents) {
+        this.registerInteractionHandler(new TouchInteractionHandler());
+      }
```

Comment:
do 'pointer' and 'touch' events conflict in ways that 'touch' and 'mouse' events don't?

Reply:
Yeah, if pointer events are supported, there's no need to capture touch events.